apps/framework-internal-app/project.toml

```diff
@@ -3,7 +3,7 @@ language = "Typescript"
 [redpanda_config]
 broker = "localhost:19092"
 message_timeout_ms = 1000
-retention_ms = 30000
+retention_ms = 21600000
```

Comment:
Bumping it up for prod load test. Redpanda Cloud default is 6 hours: https://docs.redpanda.com/current/manage/cluster-maintenance/disk-utilization/#configure-message-retention